src/udiskslinuxfilesystem.c

```diff
@@ -1852,10 +1852,10 @@ handle_unmount (UDisksFilesystem      *filesystem,
     }
 
   /* if system-managed (e.g. referenced in /etc/fstab or similar) and
-   * with the option x-udisks-auth, just run umount(8) as the
+   * with the option x-udisks-auth or users, just run umount(8) as the
    * calling user
    */
-  if (system_managed && has_option (fstab_mount_options, "x-udisks-auth"))
+  if (system_managed && (has_option (fstab_mount_options, "x-udisks-auth") || has_option  (fstab_mount_options, "users")))
```

Comment:
Interesting, I wasn't aware about the `users` mount option, I've always used `user` instead. And I think this should be extended by handling the `user` option as well (unless it's handled somehow else, which I can't remember...).

Reply:
`user` allows only mounting, `users` allows both mounting and unmounting by any user. We are not checking this in `handle_mount` either so I'll fix this too.

Reply:
`user` allows also unmounting, by recording the UID that triggered the mount and checking that UID on unmount. I guess the only difference is the _"allow unmounting by anyone else"_ scenario.

Would you mind adding a test case for the `user` option as well please? The test case would be almost identical I suppose. :beers: